recipes/friendly-styles/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "friendly-styles" %}
+{% set version = "0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/friendly_styles-{{ version }}.tar.gz
+  sha256: 4babd7780f69863766773f1f7f9f0c4e2f5ba0ce124407255ba15d0dd6858c6b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pygments >=2
+    - python >=3.6
+
+test:
+  imports:
+    - friendly_styles
+  commands:
+    - pip check
+    - "'SELECT * FROM table LIMIT 10' | pygmentize -s -l SQL -O style=friendly_light"  # [unix]
+    - "'SELECT * FROM table LIMIT 10' | pygmentize -s -l SQL -O style=friendly_dark"  # [unix]
+
+  requires:
+    - pip
+    - pygments
```

Comment:
This shouldn't be needed given that it's included in `run`:
```suggestion
```

Reply:
Gotcha, removed